src/pyscaffold/api/__init__.py

```diff
@@ -206,10 +218,11 @@ def verify_options_consistency(struct, opts):
                 "Directory {dir} already exists! Use the `update` option to "
                 "update an existing project or the `force` option to "
                 "overwrite an existing directory.".format(dir=opts['project']))
-    if not utils.is_valid_identifier(opts['package']):
-        raise InvalidIdentifier(
-            "Package name {} is not a valid "
-            "identifier.".format(opts['package']))
+    else:
```

Comment:
Wouldn't here an `elif` make sense to avoid another level of indentation?

Reply:
Done!